retrofit/src/test/java/retrofit2/Java8DefaultMethodsTest.java

```diff
@@ -34,12 +38,13 @@
   //  Retrofit retrofit = new Retrofit.Builder()
   //      .baseUrl(server.url("/"))
   //      .addConverterFactory(new ToStringConverterFactory())
+  //      .validateEagerly(true)
```

Comment:
Can you move this to a separate test class? I don't want to conflate the two things being tested here together.